### ISSUE TITLE
Build Failing on Heroku Deploy Button

### ISSUE THREAD

**Issue by MHaggis** [heroku/windmill#74]
We're using the one click Heroku install and it is failing. It gets to "Run scripts & scale dynos", eventually drops "Postdeploy exit code was not 0" with this left in the scroll:

You have 15 pending migrations:
  20150813150519 CreateEndpoints
  20150814192247 AddLastConfigTimeToEndpoints
  20150817123135 AddIndexToEndpoints
  20150817205345 AddColumnsToEndpoints
  20150828181233 CreateConfigurations
  20150828185208 AddConfigurationIdToEndpoints
  20150901161403 CreateConfigurationGroups
  20150901162501 AddConfigurationGroupIdToConfigurations
  20150901191504 AddConfigurationGroupIdToEndpoints
  20150909203542 RenameEndpointConfigurationId
  20150909203739 AddLastConfigIdToEndpoints
  20150912150721 AddDefaultConfigIdToConfigurationGroup
  20151014140748 AddCanaryIdToConfigurationGroup
  20151117161131 CreateApiKeys
  20151117161731 AddIndexToApikey
Run `rake db:migrate` to update your database then try again.

Let me know if there is anything else I can provide. 
Thank you!

**Comment by byt3smith** [comment]
Looks like error is on connecting to Postgres:
```
PG::ConnectionBad: could not connect to server: Connection refused
       Is the server running on host "ec2-52-54-83-162.compute-1.amazonaws.com" (52.54.83.162) and accepting
       TCP/IP connections on port 5432?
```

**Comment by byt3smith** [comment]
This issue was resolved by #76
